### DISCUSSION TITLE
Another problematic amiga game protection, "Rock N Roll"...

### DISCUSSION THREAD

**Discussion by realmml** [keirf/disk-utilities#187] [Q&A] [answered]
Hi! :)

We have found another game that disk-image can not convert or I can not find a game with a similar protection to use as base for the IPF  conversion. In this case is the original 2 floppies version of Rock N Roll by Rainbow Arts, those original german games have nice protections.

You can download the raw dumps [here](https://www.dropbox.com/scl/fi/ld9ceapxspjrxzj6va3l6/rock-n-roll.zip?rlkey=ky7dshj1lvhcpb3j48m8txbsi&dl=0).

Thanks for taking a look! :)

**Comment by kkrellwitz** [comment]
I will take a look, just have a few decoders to finish up first.

**Comment by kkrellwitz** [comment, marked as answer]
@realmml I have created the decoder for the game.  I have a little more testing to do and some cleanup.  I have tested the game with the WHDLoad install and it installs from the generated IPFs fine.

**Comment by realmml** [comment]
Thanks Keith!!! :)

**Reply by kkrellwitz** [reply]
I just released new updates including the Rock 'n Roll decoder. Use format "Rock 'n Roll Retail"
